README.md

```diff
@@ -79,6 +79,24 @@ const config = {
 const result = await authorize(config);
 ```
 
+### `prefetchConfiguration`
+
```

Comment:
Could you add a note here to say this is Android only? (similar to how we do it in config docs)

Reply:
Yup!